torch/ao/quantization/backend_config/tensorrt.py

```diff
@@ -193,6 +220,14 @@ def get_tensorrt_backend_config_dict():
             non_weighted_op_qint8_dtype_config,
         ]
     }
+    linear_op_dtype_configs = [
+        weighted_op_qint8_dtype_config,
+        weighted_op_int8_dtype_config,
+        default_dynamic_int8_dtype_config,
+        default_dynamic_float16_dtype_config,
+        # TODO: maybe remove this since fbgemm/qnnpack doesn't have kernels for it
+        default_op_fp16_dtype_config,
```

Comment:
we can remove everything here and just keep `weighted_op_qint8_dtype_config` since that's the only thing supported by tensorrt